tor/core/users.py

```diff
@@ -17,9 +18,10 @@ class UserDataNotFound(Exception):
 
 
 class User(object):
-    """
+    """A user object.
+
     Usage:
-    from users import User
+    from users import User.
```

Comment:
Not sure why we have a period here?

```suggestion
    from users import User
```

Reply:
Honestly it doesn't really matter because this whole section needs to be removed anyway

Reply:
Removed the section.